.github/workflows/acceptance-tests.yaml

```diff
@@ -4,7 +4,7 @@ on:
   workflow_dispatch:
 
 jobs:
-  acceptance_tests:
+  test:
```

Comment:
please don't rename it, it will create a duplicate in the GH Actions history, which I would need to clean manually again because there's no button to delete all the entries from it